scripts/mkcloud

```diff
@@ -275,6 +275,14 @@ function onadmin
         time onadmin_$cmd "$@"
 }
 
+function onadmin_bg
+{
+    # functions can have parameters, so pass on all except $1
+    local cmd=$1
+    shift
+    safely sshrun onadminbg_$cmd "$@ >/dev/null 2>&1 &"
```

Comment:
Btw. does this really return as expected?
My tests with `ssh $somewhere "sleep 5 &"` show that ssh is actually waiting.